zeeguu/api/endpoints/user.py

```diff
@@ -198,3 +198,24 @@ def send_feedback():
     user = User.find_by_id(flask.g.user_id)
     ZeeguuMailer.send_feedback("Feedback", context, message, user)
     return "OK"
+
+
+@api.route("/leave_cohort/<cohort_id>", methods=["GET"])
+@cross_domain
+@requires_session
+def leave_cohort(cohort_id):
+    from zeeguu.core.model import db
+
+    """
+    set the native language of the user in session
```

Comment:
this comment is off?

Reply:
I didn't edit it, seems like it has been this way for 6 years - but I can remove it - I think it came in the diff due to the formatting.